sw-precache-config.js

```diff
@@ -9,5 +9,6 @@ module.exports = {
     '/data/**/*'
   ],
   navigateFallback: '/index.html',
+  directoryIndex: 'index.html',
```

Comment:
seems it's not necessary to specify this prop, as it goes directoryIndex: 'index.html' by default.